modules/text_server_adv/text_server_adv.cpp

```diff
@@ -845,14 +845,10 @@ _FORCE_INLINE_ TextServerAdvanced::FontTexturePosition TextServerAdvanced::find_
 
 	if (ret.index == -1) {
 		// Could not find texture to fit, create one.
-		int texsize = MAX(p_data->size.x * p_data->oversampling * 8, 256);
-
+		int texsize = MAX(p_data->size.x * p_data->oversampling * 16, 1 << (int)GLOBAL_GET("gui/fonts/min_atlas_texture_size"));
```

Comment:
How often is this method called?
`GLOBAL_GET` should not be overused, cache the value when possible (#103763 for context).

Reply:
Not extremely often, but text server already monitor project setting changes for other values, so there's no reason not to cache it too.